### PR TITLE
Don't show giftbox menu in sidebar when not using Dutch

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Added check on sidebar menu items to hide giftbox menu item if not using Dutch

***

Ticket: https://jira.uitdatabank.be/browse/III-3695

### CODE REVIEW

File: src/components/SideBar.js [hunk 1]
```diff
@@ -200,14 +200,16 @@ const NotificationMenu = memo(
           <Badge>{countUnseenAnnouncements}</Badge>
         ),
         onClick: onClickAnnouncementsButton,
+        visible: i18n.language === 'nl',
```
Comment (bertramakers) [inline src/components/SideBar.js:203]: @geoffreydhuyvetters @simon-debruijn is this okay with the `.filter()` or should we add some kind of `visible` prop on box and use that instead? I couldn't find one in Storybook so I assume it doesn't exist yet.

Comment (bertramakers) [reply src/components/SideBar.js:203]: Ah, I think I can just add a prop to `MenuItem`

Comment (bertramakers) [reply src/components/SideBar.js:208]: See https://github.com/cultuurnet/udb3-frontend/pull/176/commits/e977e6c5482d19d3cd424ff0b3277be6ade4b84b, but I can still revert that if the first solution was better.

Comment (simon-debruijn) [reply src/components/SideBar.js:208]: I prefer the last solution

**Review by simon-debruijn — APPROVED** [review]

File: src/components/SideBar.js (hunk 1 above)
Comment (bertramakers) [reply src/components/SideBar.js:208]: Do you think it would be valuable to add the `visible` prop to `Box` while we're at it? cc @geoffreydhuyvetters ?

Comment (geoffreydhuyvetters) [reply src/components/SideBar.js:208]: I was planning on doing that idd, makes sense to have it everywhere IMO.

Comment (geoffreydhuyvetters) [reply src/components/SideBar.js:208]: We can merge this one as is

**Review by geoffreydhuyvetters — APPROVED** [review]

**Pull request merged**